packages/react/src/components/task-form/TaskFormDetails.tsx

```diff
@@ -47,105 +46,112 @@ const renderStatusField = (data: string) => (
-          <SimpleItem cssClass='accent'>
-            <FormTextbox
-              label='Company'
-              value={data.company}
-              isEditing={!editing}
-              onValueChange={updateField('company')}
-            />
-          </SimpleItem>
-          <SimpleItem cssClass='accent'>
-            <FormTextbox
-              label='Assigned to'
-              value={data.owner}
-              isEditing={!editing}
-              onValueChange={updateField('owner')}
-            />
-          </SimpleItem>
-          <SimpleItem>
-            <SelectBox
-              label='Priority'
-              value={data.priority}
-              items={PRIORITY_ITEMS}
-              readOnly={!editing}
-              stylingMode='filled'
-              fieldRender={renderPriorityField}
-              itemRender={renderPriorityItem}
-              onValueChange={updateField('priority')}
-            />
-          </SimpleItem>
-          <SimpleItem>
-            <SelectBox
-              label='Status'
-              value={data.status}
-              items={STATUS_ITEMS}
-              readOnly={!editing}
-              stylingMode='filled'
-              fieldRender={renderStatusField}
-              itemRender={renderStatusItem}
-              onValueChange={updateField('status')}
-            />
-          </SimpleItem>
-          <SimpleItem>
-            <DateBox
-              value={data.startDate}
-              readOnly={!editing}
-              name='Set Start Date'
-              label='Start Date'
-              inputAttr={{ class: 'form-editor-input' }}
-              stylingMode='filled'
-              placeholder='MM/dd/y'
-              displayFormat='MM/dd/y'
-              onValueChange={updateField('startDate')}
-            >
-              <Validator>
-                <RequiredRule />
-              </Validator>
-            </DateBox>
-          </SimpleItem>
-          <SimpleItem>
-            <DateBox
-              value={data.dueDate}
-              readOnly={!editing}
-              name='Set Due Date'
-              label='Due Date'
-              inputAttr={{ class: 'form-editor-input' }}
-              stylingMode='filled'
-              placeholder='MM/dd/y'
-              displayFormat='MM/dd/y'
-              onValueChange={updateField('dueDate')}
-            >
-              <Validator>
-                <RequiredRule />
-              </Validator>
-            </DateBox>
-          </SimpleItem>
-        </GroupItem>
-
-        <SimpleItem colSpan={2}>
-          <TextArea
-            label='Details'
+    <Form
+      labelMode='floating'
+      className={classNames({ 'plain-styled-form': true, 'view-mode': !editing })}
+      validationGroup='taskValidationGroup'
+    >
+      <SimpleItem visible={!data.text}>
+        <FormTextbox
+          label='Subject'
+          value={data.text}
+          isEditing={!editing}
+          onValueChange={updateField('text')}
+        ></FormTextbox>
```

Comment:
Use self-closing tags for components without children

Reply:
Fixed

Reply:
Fixed